client/src/ingredients/error-boundary.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+/**
+ * The error boundary for ingredients.
+ *
+ * When an ingredient crashes, for whatever reason,
+ * this component will show a friendly message
+ * to replace that crashed component
+ */
+export class ErrorBoundary extends React.Component {
+  state = {
+    error: null
+  };
+  componentDidCatch(error, _errorInfo) {
+    this.setState({
+      error
+    });
+    // TODO: Report this error somewhere?

```

Comment:
Put in a comment that links to https://github.com/mdn/stumptown-renderer/issues/99